src/app/server/AppDelegate.h

```diff
@@ -29,11 +29,26 @@ class AppDelegate
 public:
     virtual ~AppDelegate() {}
     /**
-     * This is called on start of session establishment process
+     * This is called when the PBKDFParamRequest is received and indicates the start of the session establishment process
      */
     virtual void OnCommissioningSessionEstablishmentStarted() {}
+
+    /**
+     * This is called when the commissioning session has been established
+     */
     virtual void OnCommissioningSessionStarted() {}
-    virtual void OnCommissioningSessionStopped(CHIP_ERROR err) {}
```

Comment:
Can you actually change this API? Doesn't this break any consumers that were listening to it?

Reply:
This API was changed in a PR late last week to include the CHIP\_ERROR param. See [https://github.com/project-chip/connectedhomeip/pull/28138/files#r1271221619](https://github.com/project-chip/connectedhomeip/pull/28138/files#r1271221619)

It did not have that param before that, so this PR is effectively reverting it back to the way the API used to be,